### PR TITLE
Some changes to broadcasting

### PR DESCRIPTION
**Pull request by ihatethisengine**
# About the pull request
I was writing this thing and accidentally closed my browser, so I'll be brief.
1) You can use the camera with one hand. You still need to turn it on. It keeps broadcasting even if dropped, but loses range. Xeno can turn the camera off by slashing it.
2) You can unanchor TV with a wrench.
3) Broadcasting camera cant take photos anymore. But photo camera can be used with one hand

# Explain why it's good for the game
1) More convenient.
2) If no MT is around you can still have an audience.
3) I don't think it was very convenient anyway, just use reg camera for photos if you wanna.
# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl: ihatethisengine
add: You can use the camera with one hand. You still need to turn it on. It keeps broadcasting even if dropped, but loses range. Xeno can turn the camera off by slashing it.
add: You can unanchor TV with a wrench
/:cl:

### CODE REVIEW

**Comment by thevandie** [comment]
I pretty much always carried a default camera as a backup anyways, so yeah good change 👍

**Comment by cuberound** [comment]
eh kinda sad that you can not take pictures, this makes creating fax news harder then curent camera as you have to drop cammera to take pictures

**Comment by ihatethisengine** [comment]
> eh kinda sad that you can not take pictures, this makes creating fax news harder then curent camera as you have to drop cammera to take pictures

I can just make reg camera useable with one hand if this is such a big deal

File: code/game/machinery/computer/camera_console.dm [hunk 1]
```diff
@@ -243,12 +244,44 @@
...
+		toggle_anchored(wrench, user)
+		return TRUE
+	..()
+
+/obj/structure/machinery/computer/cameras/wooden_tv/broadcast/toggle_anchored(obj/item/wrench, mob/user)
+	. = ..()
+	if(!.)
+		return
+	if(!anchored)
+		src.stat |= MAINT
```
Comment (harryob) [inline code/game/machinery/computer/camera_console.dm:263]: src.stat -> stat

File: code/game/machinery/computer/camera_console.dm [hunk 2]
```diff
@@ -243,12 +244,44 @@
...
+		clear_camera()
+		current = null
+		SEND_SIGNAL(src, COMSIG_CAMERA_CLEAR)
+	else
+		src.stat &= ~MAINT
```
Comment (harryob) [inline code/game/machinery/computer/camera_console.dm:268]: ditto

File: code/game/machinery/computer/camera_console.dm [hunk 3]
```diff
@@ -243,12 +244,44 @@
...
+	update_icon()
+
+/obj/structure/machinery/computer/cameras/wooden_tv/broadcast/update_icon()
+	. = ..()
+	if(src.stat & BROKEN)
+		return
+	if(src.stat & MAINT)
```
Comment (harryob) [inline code/game/machinery/computer/camera_console.dm:275]: ditto

**Review by harryob — CHANGES_REQUESTED** [review]

**Review by harryob — APPROVED** [review]

**Pull request merged**